lib/ae_mdw/sync/async_tasks/update_aex9_presence.ex

```diff
@@ -15,27 +17,40 @@ defmodule AeMdw.Sync.AsyncTasks.UpdateAex9State do
   require Logger
 
   @microsecs 1_000_000
-  @min_height 500_000
 
   @spec process(args :: list()) :: :ok
   def process([contract_pk]) do
     Log.info("[update_aex9_state] #{inspect(contract_pk)} ...")
-    {_type, height, _hash} = height_hash = DBN.top_height_hash(false)
+    {{kbi, mbi} = block_index, call_txi} = get_call_bi_and_txi(contract_pk)
 
-    if height > @min_height do
```

Comment:
removed this pre-condition once it's difficult to discriminate the reason of missing records.